forge-core/src/main/java/forge/card/CardEdition.java

```diff
@@ -561,7 +563,37 @@ public List<PrintSheet> getPrintSheetsBySection() {
             PrintSheet sheet = new PrintSheet(String.format("%s %s", this.getCode(), section.getKey()));
 
             for (EditionEntry card : section.getValue()) {
-                sheet.add(cardDb.getCard(card.name, this.getCode(), card.collectorNumber));
+                PaperCard cardFound = null;
+
+                if (card.name.indexOf(NameSetSeparator) > -1) {
+                    // Card name is a query so ignore CollectorNumber even if present
+                    var split = card.name.split("\\" + NameSetSeparator);
+
+                    var cardName = split[0].trim();
+                    var setCode = split[1].trim();
+
+                    if (split.length > 2) {
+                        // Search within a set with ArtIndex
+                        try {
+                            var artIndex = Integer.parseInt(split[2]);
+                            cardFound = cardDb.getCard(cardName, setCode, artIndex);
+                        } catch (NumberFormatException  e) {
+                            // Invalid ArtIndex so ignore
+                            cardFound = cardDb.getCard(cardName, setCode);
```

Comment:
I don't like the ArtIndex there,
can't we use `Sword of Fire and Ice|SPG|62` or something to have the right Collector Number?
(or might need to change how EditionEntry is parsed?)